modules/ROOT/pages/edi-ack-reconciliation.adoc

```diff
@@ -1,19 +1,21 @@
 = EDI Acknowledgment Reconciliation
 
-As your trading partners acknowledge your messages, Partner Manager automatically updates their acknowledgement status. If you want to be notified, you must specifically configure acknowledgement settings when you create the message.
+As your partners acknowledge your messages, Partner Manager automatically updates their acknowledgement status. If you want to be notified, you must specifically configure acknowledgement settings when you create the message.
```

Comment:
To be notified, you must specifically configure acknowledgment settings when you create the message.

Reply:
OK